plugins/kiali/src/pages/TrafficGraph/TrafficGraphCard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import '@backstage/core-components';
+
+import { useEntity } from '@backstage/plugin-catalog-react';
+
+import { Card, CardHeader } from '@material-ui/core';
+
+import { ENTITY } from '../../types/types';
+import TrafficGraphPage from './TrafficGraphPage';
+
+export const TrafficGraphCard = () => {
+  const { entity } = useEntity();
+
+  return (
+    <Card style={{ marginRight: 20, marginBottom: 20, height: 425.59 }}>
```

Comment:
Why is this margin right? I think it is not looking good for the Kiali tab:

![image](https://github.com/user-attachments/assets/fb8967e1-35b6-43b2-8416-8c6807a4be7a)